x-pack/plugins/triggers_actions_ui/public/application/alerts_app.tsx

```diff
@@ -29,23 +27,18 @@ export const renderApp = (deps: TriggersAndActionsUiServices) => {
 };
 
 export const App = ({ deps }: { deps: TriggersAndActionsUiServices }) => {
-  const { dataViews, theme, theme$ } = deps;
-  const isDarkMode = theme.getTheme().darkMode;
+  const { dataViews, i18n, theme } = deps;
 
   setDataViewsService(dataViews);
   return (
-    <I18nProvider>
```

Comment:
The i18n context provider is not explicitly needed since KibnaRenderContextProvider provides it